lib/pf/Switch/Meraki/AP_http_CoA.pm

```diff
@@ -0,0 +1,383 @@
+package pf::Switch::Meraki::AP_http_CoA;
```

Comment:
Not really a good name...

Something more along the : AP_http_vXXX where XXX would be the version number at which Meraki starts supporting this.

Now I guess AP_http_v2 would do